src/cmd/list.go

```diff
@@ -217,14 +218,20 @@ func listOutput(images, containers []map[string]interface{}) {
 	}
 
 	if len(containers) != 0 {
+		// Because text/tabwriter does not support escape sequences for colors, we have to do some
+		// shenanigans (string of the same length; adding color to every line) to make this work
+		noColor := "\033[0m" // No color
+		color := "\033[0;00m"
```

Comment:
Terminal escape sequences are like voodoo, so I wonder if we can add some comment or reference explaining the SGR sequences.

If I understood correctly, your clever hack was to use `\033[0;00m`, instead of the more idiomatic `\033[0m`, to match the number of characters in `\033[1;32m`. Right? I think it would be good to mention this in the commit message, because most documentation on SGR sequences will mention `\033[0m` and readers will be left confused what the other thing means.

Reply:
I took the liberty to move most of the explanation into the Git commit message, and added some extra text about the matching lengths.

(I generally prefer the commit message over code comments. The former is set in stone and closely associated with code changes, while the latter can slowly drift out of sync as the surrounding code changes and people forget to update the comment.)